packages/imperative/src/plugins/AbstractPluginLifeCycle.ts

```diff
@@ -0,0 +1,51 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+/**
+ * An optional module of a plugin extends this class to
+ * perform actions during the installation lifecycle of a plugin.
+ * A plugin identifies that it supplies a life cycle class
+ * by specifying the 'pluginLifeCycle' imperative configuration property.
+ */
+export abstract class AbstractPluginLifeCycle {
+    /**
+     * This function will be called after a plugin is installed.
+     * A plugin can use this opportunity to perform a sanity test or to
```

Comment:
is "sanity test" referenced here related to the deprecated `pluginHealthCheck`?

Reply:
Potentially yes. See the earlier comment.